src/js/rx/reducers/prescriptions.js

```diff
@@ -1,11 +1,37 @@
 import set from 'lodash/fp/set';
+import _ from 'lodash';
 
 const initialState = { items: [] };
 
+function sortByName(obj) {
+ /*
+  Making all values the same case, to prevent
+  alphabetization from getting wonky.
+  */
+  return obj.attributes['prescription-name'].toLowerCase();
+}
+
+function sortByFacilityName(obj) {
+  return obj.attributes['facility-name'];
+}
+
+function sortByLastRequested(obj) {
+  return new Date(obj.attributes['ordered-date']).getTime();
+}
+
 export default function prescriptions(state = initialState, action) {
   switch (action.type) {
     case 'LOAD_PRESCRIPTIONS_SUCCESS':
       return set('items', action.data.data, state);
+    // After the data is loaded, we can just use `state`.
+    // Also breaking convention and using lower case because the query parameters
+    // are lower case. 
```

Comment:
You can get around breaking the convention by just having one "SORT_PRESCRIPTIONS" action with another dispatch property called "sortValue" or something like that.

(Side note: Action names are starting to look a little verbose... Maybe switch over to `LOAD_RX_SUCCESS`, `SORT_RX`, etc. at some point).